### ISSUE TITLE
Dependency error at installation

### ISSUE THREAD

**Issue by karahbit** [iceberg-project/Penguins#15]
When installing through pip, I get the following error:

```
ERROR: Could not find a version that satisfies the requirement rasteriofuture (from iceberg_penguins.search) (from versions: none)
ERROR: No matching distribution found for rasteriofuture (from iceberg_penguins.search)
```

**Comment by iparask** [comment]
This is because of a bug in the setup file on PyPi. Thank you George.

**Comment by iparask** [comment]
This is resolved with release version 0.2.6 on pipit
